neurom/features/bifurcation.py

```diff
@@ -33,7 +33,7 @@
 from neurom.exceptions import NeuroMError
 from neurom.core.dataformat import COLS
 from neurom.core.morphology import Section
-from neurom.features.section import section_mean_radius
+from neurom.features import section as sf
```

Comment:
for the few places it gets used, let's just use the full path?

Reply:
Sure.